cpp/src/grammar.peg

```diff
@@ -31,15 +31,13 @@ inline_lit_str     <- '"' $body< (variable / escaped / !'"' .)* > '"'
 
 escaped            <- '\\' .
 
-tag_statement      <- tag $identifier< id_name? > $classes< class_name* > space* $body< text_content? > $attributes< attr_list? >
+tag_statement      <- tag $identifier< id_name? > $classes< class_name* > ~space* $body< text_content? > $attributes< attributes? >
 
 id_name            <- '#' $name< ([a-zA-Z_] / '-')+ >
 
-class_name         <- '.' $name< ([a-zA-X_] / '-')+ >
+class_name         <- '.' $name< ([a-zA-Z_] / '-')+ >
```

Comment:
It ain't me tho 😂👌

![image](https://user-images.githubusercontent.com/8942499/28493971-efbd122c-6ed5-11e7-897f-acfba426ff80.png)